usr/share/rear/build/GNU/Linux/400_copy_modules.sh

```diff
@@ -1,28 +1,129 @@
 # 400_copy_modules.sh
 #
-# copy kernel modules for Relax-and-Recover
+# Copy kernel modules to the rescue/recovery system.
+
 #
 # This file is part of Relax-and-Recover, licensed under the GNU General
 # Public License. Refer to the included COPYING for full text of license.
 
-Log "Collecting modules for kernel version $KERNEL_VERSION"
-MODFILES=(
-$( ResolveModules "$KERNEL_VERSION" "${MODULES[@]}" "${MODULES_LOAD[@]}"  )
```

Comment:
Is ResolveModules now obsolete? Then please remove it. However, I would suggest to extract such things into functions if it can be tested independently of the other script or used elsewhere.